app/services/scene-collections/scene-collections.ts

```diff
@@ -668,16 +670,12 @@ export class SceneCollectionsService extends Service implements ISceneCollection
     if (this.autoSavePromise) await this.autoSavePromise;
   }
 
-  private async setActiveCollection(id: string) {
+  private async setActiveCollection(id: string, informServer = true) {
     const collection = this.collections.find(coll => coll.id === id);
 
     if (collection) {
       if (collection.serverId && this.userService.isLoggedIn) {
-        try {
-          await this.serverApi.makeSceneCollectionActive(collection.serverId);
-        } catch (e) {
-          console.warn('Failed setting active collection');
-        }
```

Comment:
Is there a reason for removing the `try/catch` here?

Reply:
After removing "await" keyword it doesn't make sense. I'll add ".catch()" instead

Reply:
Oh of course.  Wasn't reading the change correctly.  ty.